src/analysisd/analysisd.c

```diff
@@ -213,6 +213,9 @@ static pthread_mutex_t hourly_firewall_mutex = PTHREAD_MUTEX_INITIALIZER;
 /* Accumulate mutex */
 static pthread_mutex_t accumulate_mutex = PTHREAD_MUTEX_INITIALIZER;
 
+static pthread_mutex_t current_time_mutex = PTHREAD_MUTEX_INITIALIZER;
+static pthread_mutex_t time_ignored_mutex = PTHREAD_MUTEX_INITIALIZER;
```

Comment:
Same here.
You are not protecting all uses of rule ignore.
I think it would be better to remove this mutex for now and create a a new issue to fix all uses of rule ignore which are many more that these you are protecting.